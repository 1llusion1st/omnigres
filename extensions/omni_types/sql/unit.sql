select omni_types.unit();

select pg_column_size(omni_types.unit());

-- in/out
select ''::omni_types.unit;
select ''::omni_types.unit::text;

-- binary send/recv
select omni_types.unit_send(omni_types.unit());
