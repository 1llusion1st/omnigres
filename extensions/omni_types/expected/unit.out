select omni_types.unit();
 unit 
------
 
(1 row)

select pg_column_size(omni_types.unit());
 pg_column_size 
----------------
              1
(1 row)

-- in/out
select ''::omni_types.unit;
 unit 
------
 
(1 row)

select ''::omni_types.unit::text;
 text 
------
 
(1 row)

-- binary send/recv
select omni_types.unit_send(omni_types.unit());
 unit_send 
-----------
 \x
(1 row)

